### ISSUE TITLE
New Meter creation fails at first time

### ISSUE THREAD

**Issue by loganathan-s** [energychain/BusinessObject-MeterPointOperation#9]
Whenever I create a new meter point address using the below command, I'm getting the weird error. However, the command works perfectly for the second time

**First Try**

```
LSELLAPPA-MBP:projects loganathans$ mkdir testmeter
LSELLAPPA-MBP:projects loganathans$ cd testmeter/
LSELLAPPA-MBP:testmeter loganathans$ stromdao-mp store loganmeter 100
Initializing node: 0xd46CC3482B0c8D5245122F6Ed5075d8e6c61c56d
Balancing Contract: 0xd93F6Bb0cEA8a8cF48555B209A02209bA889B4ad
(node:63027) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): TypeError: Cannot read property 'toLowerCase' of undefined
(node:63027) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit` 
```

**Second Try**

```
LSELLAPPA-MBP:testmeter loganathans$ stromdao-mp store loganmeter 100
Initializing node: 0xd46CC3482B0c8D5245122F6Ed5075d8e6c61c56d
Balancing Contract: 0xd93F6Bb0cEA8a8cF48555B209A02209bA889B4ad
TX: 0x0190efe5cb19af95a84984a1f88c1e3fb6395221418cb97907878fc33a7118c7

```

**Comment by zoernert** [comment]
Mhmm... I can not confirm this behavior with the current released version.

```
stromhaltig:~/workspace (master) $ mkdir test
stromhaltig:~/workspace (master) $ cd test
stromhaltig:~/workspace/test (master) $ stromdao-mp store loganmeter 100
Initializing node: 0x8647B341244EbDb13EeB0FC345D5B830b3dF5809
Balancing Contract: 0x89A23F4f06687Efd0CB8425F65BeC2eD29131df3
Mandated  loganmeter 0x776a015f2c6bc582b6d1d1353f44277d7d5cd2c51d1b6a6fd6808cf840f2cebf
TX: 0xf039cb7805437f98bb83616c4e8f15ffc9f27f11df658f192a3a0fe430f62ea5
stromhaltig:~/workspace/test (master) $ 
```

Have you updated your MP tool?
Try: `npm install -g stromdao-bo-mpo`

Please report results. #Thanks

**Comment by zoernert** [comment]
Fixed in stromdao-bo-mpo@0.2.38
